src/domain/community/user/userContributions/useUserContributions.ts

```diff
@@ -23,19 +23,23 @@ const useUserContributions = (userNameId: string) => {
     }));
 
     const subspaces = data?.rolesUser.spaces.flatMap<ContributionItem>(e =>
-      e.subspaces.map(c => ({
-        spaceId: e.id,
-        challengeId: c.id,
-        id: c.id,
-      }))
+      e.subspaces
+        .filter(c => c.type === 'CHALLENGE')
+        .map(c => ({
+          spaceId: e.id,
+          challengeId: c.id,
+          id: c.id,
+        }))
     );
 
     const subsubspaces = data?.rolesUser.spaces.flatMap<ContributionItem>(e =>
```

Comment:
Can we iterate over `data?.rolesUser.spaces` just once and do the assignment into spaces, subspaces, subsubspaces there?